pkg/static/login.js

```diff
@@ -1026,24 +1030,22 @@ function debug(...args) {
                         } else {
                             // but only once, to avoid loops
                             debug("send_login_request(): invalid-hostkey, and already retried, giving up");
-                            host_failure(_("Refusing to connect. Hostkey does not match"));
+                            host_failure(_("Refusing to connect"), _("Hostkey does not match"));
                         }
                     } else if (is_conversation) {
                         login_failure(_("Authentication failed"));
                     } else {
-                        login_failure(_("Wrong user name or password"));
+                        login_failure(_("Authentication failed"), _("Wrong user name or password"));
                     }
                 }
             } else if (xhr.status == 403) {
-                login_failure(_(decodeURIComponent(xhr.statusText)) || _("Permission denied"));
+                const status = decodeURIComponent(xhr.statusText).trim();
+                login_failure(_("Permission denied"), status === "Permission denied" ? "" : status);
             } else if (xhr.status == 500 && xhr.statusText.indexOf("no-cockpit") > -1) {
-                // always show what's going on
-                let message = format(_("A compatible version of Cockpit is not installed on $0."), login_machine || "localhost");
-                // in beiboot mode we get some more info
-                const error = JSON.parse(xhr.responseText);
-                if (error.supported)
-                    message += " " + format(_("This is only supported for $0 on the target machine."), error.supported);
-                login_failure(message);
+                login_failure(
+                    _("Cockpit not installed on remote host"),
+                    format(_("Install at least the cockpit-system package on $0 to connect."), login_machine || "localhost")
```

Comment:
@garrett: I haven't quite made up my mind between "tell the user which remote OSes are supported" and "tell the user how to fix it" (install cockpit-system). "you can remotely connect to Fedora 40, Debian testing, and RHEL 9.6" seems "nice" at some level, but not in this particular situation: you want to connect to the given machine, not a random one which happens to have a compatible OS.

So I think like this it's more helpful. WDYT?

Reply:
What about something like this?

## Dynamic remote session unavailable

This system does not support dynamic remote sessions. Install the `cockpit-system` package on the remote system to manage it via the web console.

Reply:
summary from chat: so far my favourites are:

 (1) drop the first sentence and only leave the second
 (2) s/dynamic/package-less/ , as that's more descriptive of what this is about

Reply:
Alternative attempt:

## Packageless session unavailable

Transient packageless connections currently require $0. Install the `cockpit-system` package on the remote system to manage it via the web console.

Reply:
Looks like this now:

![Screen Shot 2024-10-23 at 12 02 10](https://github.com/user-attachments/assets/68bb913f-de8c-48c6-859f-83e6cb1c5406)

Reply:
The "at least" part really isn't good. It sounds super awkward.

Packageless session unavailable

Transient packageless sessions require the same operating system and version, Fedora Linux 40, for compatibility reasons. Install the `cockpit-system` package (and optionally other `cockpit` packages) on `10.111.113.2` to enable web console access. 

or

Packageless session unavailable

Transient packageless sessions require the same operating system and version, for compatibility reasons: Fedora Linux 40. Install the `cockpit-system` package (and optionally other `cockpit` packages) on `10.111.113.2` to enable web console access. 

(BTW: It's Fedora Linux 40, not "fedora 40". Note the capitalization and "Linux" in the official name.)

Also: I wonder if it's too dense and covers different topics and if each sentence should be its own paragraph? Or if we should just keep the alerts headline plus body.

Reply:
OK, thanks! Changed to your second version:

![Screen Shot 2024-10-24 at 08 50 13](https://github.com/user-attachments/assets/0b76e917-0b42-498c-bebe-d3abc2d09270)

As for the "paragraph or not": This would currently require bigger changes. But I'm going to enable it for multiple OSes as the next step anyway, and then it'll need to become a list. I suggest to postpone the paragraph splitting until then, is that ok?

Reply:
Sure! This is already way better than what we have in main. The spacing and alignment is better than in main too :wink:, even if we agree it should be improved also. This is good enough to merge, IMHO, and we can do the rest in a follow-up.